go/pkg/yolosvc/driver_github.go

```diff
@@ -187,6 +195,48 @@ func (worker *githubWorker) fetchBaseObjects(ctx context.Context) (*yolopb.Batch
 	return batch, nil
 }
 
+func getOverrideBuildpb(owner string, repo string, artiID int64, token string) (*yolopb.OverrideBuild, error) {
+	override := &yolopb.OverrideBuild{}
+	// FIXME: is it always .zip files ?
```

Comment:
yes it is: https://github.com/actions/upload-artifact#zipped-artifact-downloads

please, update the code to add a comment pointing to that link